src/cache/precomputed_quantities.jl

```diff
@@ -54,6 +54,7 @@ function precomputed_quantities(Y, atmos)
         ᶜh_tot = similar(Y.c, FT),
         sfc_conditions = Fields.Field(SCT, Spaces.level(axes(Y.f), half)),
     )
+    cloud_diagnostics = (; cloud_fraction = similar(Y.c, FT),)
```

Comment:
```suggestion
    cloud_diagnostics = (; ᶜcloud_fraction = similar(Y.c, FT),)
```
(not very important, but it might be good to specify that it is at cell center)